.gitignore

```diff
@@ -0,0 +1,7 @@
+# Mac
+.DS_STORE
+
+# Node
+node_modules/
+dist/
+.npmrc
```

Comment:
Right now `dist/` won't be published. We need something like https://github.com/FormidableLabs/trace-deps/blob/master/.npmignore (which is a reverse "include only what we want" version)

Reply:
The `npm pack` command is great for giving you a preview of what will publish for real.

Reply:
`/dist` is generated in the [GitHub workflow](https://github.com/FormidableLabs/dogs/blob/typescript-package/.github/workflows/gpr-publish.yaml#L20) via `npm run build`

[Example](https://github.com/FormidableLabs/dogs/runs/1260574891?check_suite_focus=true#step:5:9)